libstorage/MemoryTable.h

```diff
@@ -408,7 +408,7 @@ class MemoryTable : public Table
     {
         std::vector<size_t> indexes;
         indexes.reserve(entries->size());
-        if (condition->getConditions()->empty())
+        if (condition->getConditions()->empty() && !condition->isDirty())
```

Comment:
Can we do this in another way？

Reply:
I move the limit logic to CRUDPrecompiled